include/pm_string.h

```diff
@@ -5,37 +5,37 @@
 #include "heap.h"
 #include "string_util.h"
 
-typedef struct String {
+typedef struct Strbuf {
```

Comment:
keep it string please

Reply:
Strbuf is literally correct here and we went over this months ago.

Reply:
and we agreed you can keep it strbuf in plat (your repo) and we're keeping it string here